tests/test_pr.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+import torch
+
+from piq import PR
+
+
+@pytest.fixture(scope='module')
+def features_y_normal() -> torch.Tensor:
+    return torch.rand(1000, 20)
+
+
+@pytest.fixture(scope='module')
+def features_x_normal() -> torch.Tensor:
+    return torch.rand(1000, 20)
+
+
+@pytest.fixture(scope='module')
+def features_x_beta() -> torch.Tensor:
+    m = torch.distributions.Beta(torch.FloatTensor([2]), torch.FloatTensor([2]))
+    return m.sample([1000, 20]).squeeze()
+
+
+@pytest.fixture(scope='module')
+def features_x_constant() -> torch.Tensor:
+    return torch.ones(1000, 20)
+
+
+def test_initialization() -> None:
+    try:
+        PR()
+    except Exception as e:
+        pytest.fail(f"Unexpected error occurred: {e}")
+
+
+def test_forward(features_y_normal, features_x_normal, ) -> None:
+    try:
+        metric = PR()
+        metric(features_y_normal, features_x_normal)
+    except Exception as e:
+        pytest.fail(f"Unexpected error occurred: {e}")
```

Comment:
No need to catch exceptions, pytest will do it.
We have similar code in other metrics, but will fix this eventually. 
```suggestion
def test_forward(features_y_normal, features_x_normal, ) -> None:
        metric = PR()
        metric(features_y_normal, features_x_normal)
```

Reply:
You're right, I just copied KID tests without much thought as they'd technically be the same